x-pack/legacy/plugins/infra/public/pages/logs/log_entry_rate/page_results_content.tsx

```diff
@@ -259,6 +265,8 @@ const getBucketDuration = (startTime: number, endTime: number) => {
 // This is needed due to the flex-basis: 100% !important; rule that
 // kicks in on small screens via media queries breaking when using direction="column"
 export const ResultsContentPage = euiStyled(EuiPage)`
+  flex: 1 0 0%;
+
   .euiFlexGroup--responsive > .euiFlexItem {
     flex-basis: auto !important;
   }
```

Comment:
I'm sorry, I don't get why we need this.

Reply:
not sure, @Kerry350 do you remember what this was about?